zone/client_packet.cpp

```diff
@@ -16533,3 +16644,306 @@ void Client::ReloadExpansionProfileSetting()
 		m_pp.expansions = RuleI(World, ExpansionSettings);
 	}
 }
+
+void Client::Handle_OP_GuildTributeSelect(const EQApplicationPacket* app)
+{
+	LogTribute("Received OP_GuildSelectTribute of length [{}]", app->size);
```

Comment:
We get this logging for free with our existing packet logging, can remove

Reply:
Removed.